python/fledge/common/iprpc.py

```diff
@@ -235,23 +241,35 @@ def serve(self):
         """
 
         while True:
+            # eprint("RECEIVE: {}".format(str(_obj)[:200]))
+            try:
+                _obj = self.rpc_read()
+
```

Comment:
better if newline avoided here :]

Reply:
I need to start running the linter you guys use. I'm trying to resist my preference to have more whitespace to highlight things, but my instincts fail me sometimes :)